tests/wpt/runner/expectation.json

```diff
@@ -6842,52 +6852,8 @@
         "request-headers-case.any.worker.html": false,
         "request-headers-nonascii.any.html": true,
         "request-headers-nonascii.any.worker.html": true,
-        "request-headers.any.html": [
-          "Fetch with PUT without body",
-          "Fetch with PUT with body",
-          "Fetch with POST without body",
-          "Fetch with POST with text body",
-          "Fetch with POST with FormData body",
```

Comment:
This looks concerning - do you know what happened with the passing tests?

Reply:
It might be that tests were added that include `Float16Array`, which doesn't appear implemented yet. This is the error we were getting before:
```
asher@Ashers-MacBook-Air deno % deno run -A tests/wpt/wpt.ts run -- "/fetch/api/basic/request-headers.any.html"
    Finished dev [unoptimized + debuginfo] target(s) in 0.80s
Going to run 1 test files on 8 cores.
----------------------------------------
/fetch/api/basic/request-headers.any.html

test stderr:
error: Uncaught (in promise) ReferenceError: Float16Array is not defined
    at http://web-platform.test:8000/fetch/api/basic/request-headers.any.js:57:75
    at Object.evalContext (ext:core/01_core.js:649:31)
    at file:///var/folders/72/k8dpnhm14hg9lcm16pskx6_00000gn/T/wpt-bundle-96b894b091249885.js:31:44
    at file:///var/folders/72/k8dpnhm14hg9lcm16pskx6_00000gn/T/wpt-bundle-96b894b091249885.js:35:3



file result: failed. runner failed during test (4672ms)

========================================

file failures:

        "/fetch/api/basic/request-headers.any.html"

final result: failed. 0 passed; 1 failed; 0 expected failure; total 1 (6287ms)
```

Reply:
Got it, that makes sense. They will start passing once we upgrade to V8 12.4 (https://github.com/denoland/rusty_v8/pull/1459)